src/screens/ExchangeModal.js

```diff
@@ -512,14 +514,20 @@ export default function ExchangeModal({
       logger.log('error getting the swap amount in USD price', e);
     } finally {
       analytics.track(`Submitted ${type}`, {
+        aggregator: tradeDetails?.source || '',
         amountInUSD,
-        defaultInputAsset: defaultInputAsset?.symbol ?? '',
+        inputTokenAddress: defaultInputAsset?.address || '',
+        inputTokenName: defaultInputAsset?.name || '',
+        inputTokenSymbol: defaultInputAsset?.symbol || '',
```

Comment:
mainly putting a note here since we're switching the parameters (`symbol -> inputTokenSymbol`)  to make sure analytics is aware

Reply:
do i need to change something on mixpanel to match these changes?

Reply:
i think is ok for now